test/dune

```diff
@@ -0,0 +1,9 @@
+(test
+  (name benchFind)
+  (modules benchFind)
+  (libraries fileutils))
```

Comment:
@gildor478 note that this is now a test so it's tested on appveyor. I also changed the test so it exit with error code `0` if we're not on unix and to use `$HOME` instead of `/home/gildor`.